SD-K9/src/app/pages/location-search/location-search.page.ts

```diff
@@ -89,6 +91,22 @@ export class LocationSearchPage implements OnInit {
     this._navController.navigateBack("home");
   }
 
+  getCurrentPos(){
+    let currentMapState = this._googleStore.getGoogleMapState();
+    let currentPos = currentMapState.currentpos;
+    let self = this;
+    this.latlng = {lat: currentPos.lat(), lng:  currentPos.lng()}; 
+
+    self.enterQuery(' '); 
+    currentMapState.geocoder.geocode({'location': this.latlng}, function(results, status) 
```

Comment:
I think that you are not using the "geocoder" variable that you declared above, but rather the state variable in this case.

Reply:
please verify the changes in the new commit

Reply:
Looks good!